src/pages/dialog/components/MessageItem.module.css

```diff
@@ -0,0 +1,53 @@
+img {
+	border-radius: 50%;
+	width: 50px;
+	height: 50px;
+}
+
+
+.YourMessage {
+	max-width: 40vw;
+	display: flex;
+	position: relative;
+	left: 5vw;
+}
+.YourMessageRightBlock {
+	display: flex;
+	flex-direction: column;
+}
+.YourMessageRightBlock span {
+	text-align: start;
+	margin-left: 20px;
+}
+.YourMessageRightBlock p {
+	max-width: 30vw;
+	margin: 5px 0px 5px 5px;
+	padding: 5px;
+	background-color: white;
+	border: 2px solid #f1f3f4;
+	border-radius: 10px 10px 10px 0px;
+}
+
+
+.NotYourMessage {
```

Comment:
For fix design change this to 
max-width: 40vw;
display: flex;
margin-right: 5vw;
align-self: end;